agent/api/logging.go

```diff
@@ -66,7 +66,7 @@ func (lm *loggingMiddleware) Result(ctx context.Context, consumer string) (respo
 	return lm.svc.Result(ctx, consumer)
 }
 
-func (lm *loggingMiddleware) Attestation(ctx context.Context, reportData []byte) (response []byte, err error) {
+func (lm *loggingMiddleware) Attestation(ctx context.Context, reportData [size64]byte) (response []byte, err error) {
```

Comment:
what is this constant `size64`? Where does it come from?

Reply:
report data is 64 bytes long, https://github.com/google/go-sev-guest/blob/08d92a6f9da9dcab1ca702485aff381b5d3efdf9/proto/sevsnp.proto#L38

Reply:
This is OK, but is this some constant, and can we please declare it then at the beginning of the file, like `att_report_size` or something like this

Reply:
we use it a number of times so I'll set an exported constant